tox.ini

```diff
@@ -43,7 +43,8 @@ deps =
     dockerfile: ansible>=2.9.12
     selinux
 commands =
-    ansibledevel: ansible-galaxy install git+https://github.com/ansible-collections/community.general.git
+    ansibledevel: ansible-galaxy collection install git+https://github.com/containers/ansible-podman-collections
+    ansible-galaxy collection install containers.podman
```

Comment:
I'd stick it to specific version like it's done above: `require_collection("containers.podman", "1.6.1")`

Reply:
I will do that but only for the non-devel install path, so we endup testing both minimal version we require but also master branch, so we know if newer versions breaks us.